modules/test_user_dashboard/app/serializers/test_user_dashboard/tud_account_serializer.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module TestUserDashboard
+  class TudAccountSerializer < ActiveModel::Serializer
+    attributes :id, :account_uuid, :first_name, :middle_name, :last_name, :gender,
```

Comment:
Is this PR just laying down the groundwork and you'll utilize the serializer in another PR (I'm assuming in the TestUserDashboard::TudAccountsController)?

Reply:
Right now this is being added to format the TudAccounts data so that it's following JSON API specifications for the frontend to consume, in the future there will almost certainly be additional usage by the controller.